Dockerfile

```
@@ -0,0 +1,60 @@
+FROM rocker/r-ver:4.0.3
+
+# install git-lfs, for copying in giant files later on
+# also install python + pip + java
+RUN \
+  apt-get update && \
+  apt-get install -y \
+    curl git git-lfs python3 python3-pip openjdk-8-jdk \
+    pandoc qpdf libcurl4-openssl-dev libmagick++-6.q16-dev libudunits2-dev libpoppler-cpp-dev \
+    && \
+  git lfs install
```

It's been a while since I don't work with Docker, but I remember that if you cleaned up the APT cache it will lead to a smaller image size. Yes, here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
```suggestion
RUN \
  apt-get update && \
  apt-get install -y \
    curl git git-lfs python3 python3-pip openjdk-8-jdk \
    pandoc qpdf libcurl4-openssl-dev libmagick++-6.q16-dev libudunits2-dev libpoppler-cpp-dev \
&& rm -rf /var/lib/apt/lists/* \
&&  git lfs install
```

---

nice, I didn't know!